.github/workflows/build-images.yaml

```diff
@@ -10,42 +10,42 @@ jobs:
         with:
             package: jobs
             run-cmd: ts-build
-            tags: -t nangohq/nango-jobs:${{ github.sha }}
+            tags: -t nangohq/nango-jobs:$GITHUB_SHA ${{ github.ref == 'refs/heads/master' && '-t nangohq/nango-jobs:enterprise-$GITHUB_SHA -t nangohq/nango-jobs:enterprise -t nangohq/nango-jobs:hosted-$GITHUB_SHA -t nangohq/nango-jobs:hosted' || '' }}
     nango-runner:
         uses: ./.github/workflows/push-container.yaml
         secrets: inherit
         with:
             package: runner
             run-cmd: ts-build
-            tags: -t nangohq/nango-runner:${{ github.sha }}
+            tags: -t nangohq/nango-runner:$GITHUB_SHA ${{ github.ref == 'refs/heads/master' && '-t nangohq/nango-runner:enterprise-$GITHUB_SHA -t nangohq/nango-runner:enterprise -t nangohq/nango-runner:hosted-$GITHUB_SHA -t nangohq/nango-runner:hosted' || '' }}
     nango-server-staging:
         uses: ./.github/workflows/push-container.yaml
         secrets: inherit
         with:
             package: server
             run-cmd: build:staging
-            tags: -t nangohq/nango-server:staging-${{ github.sha }}
+            tags: -t nangohq/nango-server:staging-$GITHUB_SHA
```

Comment:
It is easier to use the env var instead of the var substitution in the ternary syntax above, so I switched to use the env var everywhere for consistency

Reply:
good call